web/packages/teleport/src/features.tsx

```diff
@@ -645,7 +645,7 @@ export class FeatureClusters implements TeleportFeature {
   };
 
   hasAccess(flags: FeatureFlags) {
-    return cfg.isDashboard || flags.trustedClusters;
+    return (cfg.isDashboard || flags.trustedClusters) && !cfg.isCloud;
```

Comment:
Do we need to see the clusters page if it's a dashboard?

I think we needed this in the past because that's where the contacts were, but now that they're no longer there maybe this is no longer necessary?